python/lsst/ts/pmd/component.py

```diff
@@ -0,0 +1,104 @@
+__all__ = ["MitutoyoComponent"]
```

Comment:
Please insert the standard boilerplate here and please check your other .py files (except for oddball files such as setup.py and conf.py) for missing boilerplate.